pkg/core/mem_pool.go

```diff
@@ -129,6 +129,7 @@ func (mp MemPool) TryAdd(hash util.Uint256, pItem *PoolItem) bool {
 	mp.lock.RLock()
 	_, ok := mp.unsortedTxn[hash]
 	mp.lock.RUnlock()
+	updateMempoolMetrics(len(mp.unsortedTxn), len(mp.unverifiedTxn), mp.Count())
```

Comment:
These `len()`'s should be under the lock, but beware of locking inside the `Count()`.

Reply:
`count()` gone and now `update...()` inside of lock